packages/jest-environment-wrangler/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jest-environment-wrangler",
+  "version": "0.0.23",
+  "description": "A jest environment for Cloudflare Workers, powered by wrangler.",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "esbuild src/index.ts --outfile=dist/index.js --target=es2020 --sourcemap --platform=node --format=cjs"
```

Comment:
Can you add a `check:type` script that will run the `tsc` checks on this package too?

Reply:
good catch, adding